examples/text-classification/src/training.rs

```diff
@@ -78,17 +78,18 @@ pub fn train<B: ADBackend, D: TextClassificationDataset + 'static>(
         .metric_valid(AccuracyMetric::new())
         .metric_train_plot(LossMetric::new())
         .metric_valid_plot(LossMetric::new())
-        .with_file_checkpointer::<f32>(2)
+        .with_file_checkpointer::<burn::tensor::f16>(2, StateFormat::default())
```

Comment:
Do you need to revert back this change or does it work because serialization works for std?

Reply:
serialization works with std